lib/SILOptimizer/Utils/ConstantFolding.cpp

```diff
@@ -50,7 +50,66 @@ APInt swift::constantFoldBitOperation(APInt lhs, APInt rhs, BuiltinValueKind ID)
   }
 }
 
-APInt swift::constantFoldComparison(APInt lhs, APInt rhs, BuiltinValueKind ID) {
+APInt swift::constantFoldComparisonFloat(APFloat lhs, APFloat rhs,
+                                         BuiltinValueKind ID) {
+  bool result;
+  bool isOrdered = !lhs.isNaN() && !rhs.isNaN();
+
+  switch (ID) {
```

Comment:
Please ensure your coding style is same as in the file

Reply:
This is actually `git-clang-format`. But I can bypass that step to keep the style same.

Reply:
Well... `git-clang-format` is dumb and overall the style guide is not _simply clang-format everything_.

Reply:
> not simply clang-format everything.

Shouldn't it be? What is the benefit of using a formatter then?

Reply:
It can handle maybe 80% of cases. However, the remaining 20% of cases are about things like readability, etc.

However, in your particular case it's not configured properly. The repo's style config is explicitly said `based on LLVM`. And LLVM defaults to 2 space indent. So, something is not correct here and this is why one need to examine the output of formatter not merely commit it as-is.